### PR TITLE
Sdk6toast

### PR DESCRIPTION
**Pull request by Sdk6**

### CODE REVIEW

File: web/src/pages/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+import { Link, Redirect, routes } from '@redwoodjs/router'
 import { MetaTags } from '@redwoodjs/web'
+import { toast, Toaster } from '@redwoodjs/web/toast'
+
 import { useAuth } from 'src/auth'
 
 const HomePage = () => {
   const { currentUser, isAuthenticated } = useAuth()
-
+  toast.success(`Login successful, welcome ${currentUser.email}`)
```
Comment (R2bEEaton) [inline web/src/pages/HomePage/HomePage.jsx:9]: Here the toast is always played whenever a user visits that page. This means any time they refresh they will see the same toast. We only want it to happen right after they log in. Check this example for creating a dynamic toast, and the ability to add more readable error messages to oauth.js: https://redwoodjs.com/docs/how-to/oauth#better-error-handling

**Review by R2bEEaton — CHANGES_REQUESTED** [review]
Right now the toast shows up all the time, but it should only show up on HomePage right after a login. Similarly, it should take you back to LoginPage and play a toast with an error if the login fails. Check my other comment for details.

**Comment by Sdk6** [comment]
i changed to show toast on the first login but im not sure how a google login fails, wouldnt they just automatically go back to login? I am going to sleep now but am happy to talk more about this later.

**Comment by R2bEEaton** [comment]
What you have there looks great for showing the message on first login! As for showing login errors, you are right. Most of the login errors are caught by Google. But we get some of them.

Look in /api/src/functions/oauth/oauth.js
Line 37
Line 53

Both of these catch potential errors with the login. They currently just return an error message, but you should make them redirect back to routes.login() and show a toast with that error message.

**Comment by R2bEEaton** [comment]
Merged as part of #15 I think

**Pull request closed without merging**